e2etests/tests/unnumbered.go

```diff
@@ -0,0 +1,286 @@
+		c, err := rc.Config()
+		Expect(err).NotTo(HaveOccurred())
+		err = externalP2PContainer.UpdateBGPConfigFile(c)
+		Expect(err).NotTo(HaveOccurred())
+
+		ginkgo.By(fmt.Sprintf("apply frrconfiguration %s", "for-"+externalP2PContainer.Name))
+		cr := frrk8sv1beta1.FRRConfiguration{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "for-" + externalP2PContainer.Name,
+				Namespace: k8s.FRRK8sNamespace,
+			},
+			Spec: frrk8sv1beta1.FRRConfigurationSpec{
+				BGP: frrk8sv1beta1.BGPConfig{
+					BFDProfiles: []frrk8sv1beta1.BFDProfile{{Name: "simple"}},
+					Routers: []frrk8sv1beta1.Router{
+						{
+							ASN: infra.FRRK8sASN,
+							Neighbors: []frrk8sv1beta1.Neighbor{
+								{
+									ASN:        remoteASN,
+									Interface:  iface,
+									BFDProfile: "simple",
+									ToReceive: frrk8sv1beta1.Receive{
+										Allowed: frrk8sv1beta1.AllowedInPrefixes{
+											Mode: frrk8sv1beta1.AllowAll,
+										},
+									},
+									ToAdvertise: frrk8sv1beta1.Advertise{
+										Allowed: frrk8sv1beta1.AllowedOutPrefixes{
+											Mode: frrk8sv1beta1.AllowAll,
+										},
+									},
+								},
+							},
+							Prefixes: prefixSend,
+						},
+					},
+				},
+				NodeSelector: metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"kubernetes.io/hostname": nodeWithP2PConnection.GetLabels()["kubernetes.io/hostname"],
+					},
+				},
+			},
+		}
+		err = updater.Update([]corev1.Secret{}, cr)
+		Expect(err).NotTo(HaveOccurred())
+		ginkgo.DeferCleanup(func() {
+			err := updater.DeleteFRRConfiguration(cr)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		nodeLLA, err := container.GetIPv6Link(nodeWithP2PConnection.Name, iface)
+		Expect(err).NotTo(HaveOccurred())
+		peerLLA, err := container.GetIPv6Link(externalP2PContainer.Name, iface)
+		Expect(err).NotTo(HaveOccurred())
+		ginkgo.By(fmt.Sprintf("%s/%s/%s --- %s/%s/%s",
+			nodeWithP2PConnection.Name, iface, nodeLLA, iface, peerLLA, externalP2PContainer.Name))
+
+		validateBGPPeering(externalP2PContainer, nodeLLA)
+
+		ginkgo.By(fmt.Sprintf("validate routing table on %s (prefixSend)", externalP2PContainer.Name))
+		validatePrefixViaFor(externalP2PContainer, iface, nodeLLA, prefixSend...)
+		// we cannot use
+		// ValidatePrefixesForNeighbor(*externalP2PContainer, []corev1.Node{nodeWithP2PConnection}, prefixSend...)
+		// because deep down the LLA ip cannot be found as part of the node
+
+		ginkgo.By(fmt.Sprintf("validate routing table on %s (prefixReceive)", nodeWithP2PConnection.Name))
+		exc1 := executor.ForContainer(nodeWithP2PConnection.Name)
+		validatePrefixViaFor(exc1, iface, peerLLA, rc.ToAdvertiseV4, rc.ToAdvertiseV6)
+
+		// We can also use existing functions if fix the frr.Ipv4 to be LLA
+		pod, err := k8s.FRRK8sPodOnNode(cs, nodeWithP2PConnection.Name)
+		Expect(err).NotTo(HaveOccurred())
+		ValidateNodesHaveRoutes([]*corev1.Pod{pod}, *externalP2PContainer, rc.ToAdvertiseV4, rc.ToAdvertiseV6)
```

Comment:
what does this add?

Reply:
nothing I let it there to let you know that the usual function can work also in case you wanted to use that. Removed